lib/travis/build/script/matlab.rb

```diff
@@ -0,0 +1,46 @@
+module Travis
+  module Build
+    class Script
+      class Matlab < Script
+        MATLAB_INSTALLER_LOCATION = 'https://ssd.mathworks.com/supportfiles/ci/ephemeral-matlab/v0/install.sh'.freeze
+        MATLAB_DEPS_LOCATION = 'https://ssd.mathworks.com/supportfiles/ci/matlab-deps/v0/install.sh'.freeze
+        MATLAB_START = 'matlab -batch'.freeze
+        MATLAB_COMMAND = "assertSuccess(runtests('IncludeSubfolders',true));".freeze
+
+        DEFAULTS = {
+          matlab: 'latest'
+        }.freeze
+
+        def export
+          super
+          sh.export 'TRAVIS_MATLAB_VERSION', release.shellescape,
+                    echo: false
```

Comment:
Stylistically, you can merge this up with the preceding one.

Reply:
Done! 😄